search-solr/src/main/java/edu/unc/lib/boxc/search/solr/services/SolrSearchService.java

```diff
@@ -562,6 +562,9 @@ private String getRangeValue(String key, String value) {
         if (StringUtils.isBlank(value)) {
             return "*";
         }
+        if (SearchSettings.FIELDS_RANGE_SEARCHABLE.contains(key)) {
```

Comment:
this set also contains `SearchFieldKey.DATE_CREATED.name(), SearchFieldKey.DATE_ADDED.name(), SearchFieldKey.DATE_UPDATED.name()`, have you tested to see if those fields are still searchable in the advanced search? It seems like they would need to go to the FIELDS_DATE_SEARCHABLE block. Both FIELDS_RANGE_SEARCHABLE and FIELDS_DATE_SEARCHABLE contain the same values at the moment